Content.Server/Tabletop/TabletopBackgammonSetup.cs

```diff
@@ -0,0 +1,91 @@
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+
+namespace Content.Server.Tabletop
+{
+    [UsedImplicitly]
+    public class TabletopBackgammonSetup : TabletopSetup
+    {
+        [DataField("boardPrototype")]
+        public string BackgammonBoardPrototype { get; } = "BackgammonBoardTabletop";
+
+        [DataField("whitePiecePrototype")]
+        public string WhitePiecePrototype { get; } = "WhiteTabletopPiece";
+
+        [DataField("blackPiecePrototype")]
+        public string BlackPiecePrototype { get; } = "BlackTabletopPiece";
+
+
```

Comment:
Whitespace

Reply:
I should work on finding a linter, or just get better about these, thank you.